sdk/eventhub/event-hubs/samples-dev/receiveEvents.ts

```diff
@@ -8,19 +8,25 @@
  */
 
 import { EventHubConsumerClient, earliestEventPosition } from "@azure/event-hubs";
+import { DefaultAzureCredential } from "@azure/identity";
 
 // Load the .env file if it exists
-import * as dotenv from "dotenv";
-dotenv.config();
+import "dotenv/config";
```

Comment:
interesting.  Looks that the dotenv docs recommends this for ES6.  However, I feel that our sample publishing may not be smart enough to generate the right CJS version. as show below it was converted to just `require("dotenv/config");`

Reply:
I tested it and `require("dotenv/config");` works fine.